setup.py

```diff
@@ -9,6 +9,7 @@
     zip_safe=False,
     packages=['app'],
     install_requires=[
+        "config",
```

Comment:
I see, I see, can you dm me your `config.py`, (redacted of course) so I can play around with it. Maybe we can have the `config.py` reading from a json of a `.env` file to fill in these credentials. I'd like the error to show incorrect credentials instead of `No module`.

Reply:
```
import os

basedir = os.path.abspath(os.path.dirname(__file__))

class Config(object):
    SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://sqltable:password@hostname:port/sqltable'
    SQLALCHEMY_TRACK_MODIFICATIONS = False
    SECRET_KEY = os.environ.get('ENV_VAR') or 'password'
```

I'm only familiar with `json` but open to other formats as well. There are 2 important variables that need to be set. The key, and the database URI.

Reply:
https://github.com/stevesdawg/govstat/issues/4